LayoutTests/fast/dom/event-handler-attributes-expected.txt

```diff
@@ -88,12 +88,12 @@ PASS testScriptAttribute(window, "beforecopy") is "none"
 PASS testScriptAttribute(window, "beforecut") is "none"
 PASS testScriptAttribute(window, "beforeload") is "none"
 PASS testScriptAttribute(window, "beforepaste") is "none"
-PASS testScriptAttribute(window, "copy") is "none"
-PASS testScriptAttribute(window, "cut") is "none"
+FAIL testScriptAttribute(window, "copy") should be none. Was window.
+FAIL testScriptAttribute(window, "cut") should be none. Was window.
 PASS testScriptAttribute(window, "focusin") is "none"
 PASS testScriptAttribute(window, "focusout") is "none"
 PASS testScriptAttribute(window, "noneventname") is "none"
-PASS testScriptAttribute(window, "paste") is "none"
+FAIL testScriptAttribute(window, "paste") should be none. Was window.
```

Comment:
The test should be updated accordingly to expect window instead of none

Reply:
I updated again...

Reply:
Looks good to me, thanks!